Jenkinsfile

```diff
@@ -1,129 +1,142 @@
-properties(
-    [
-    buildDiscarder
-        (logRotator (
+properties([
+    buildDiscarder(
+        logRotator(
             artifactDaysToKeepStr: '',
             artifactNumToKeepStr: '',
             daysToKeepStr: '14',
-            numToKeepStr: '10'
-        ) ),
-    disableConcurrentBuilds()
-    ]
-)
-
+            numToKeepStr: '10',
+        )
+    ),
+    // Make new builds terminate existing builds
+    disableConcurrentBuilds(
+        abortPrevious: true,
+    )
+])
 pipeline {
-
     agent {
-        // Use the docker to assign the Python version.
-        // Use the label to assign the node to run the test.
-        // It is recommended by SQUARE to not add the label
+        // Run as root to avoid permission issues when creating files.
```

Comment:
Is this still needed? I thought the docker image we use for the unit tests has all software installed?

Reply:
Yes it seems to be. At least I tried to remove it earlier when I was working on updating Eric's template and I could not figure out how to get rid of it. I would love to do so.